js/components/contactList/contactList_controller.js

```diff
@@ -92,10 +139,8 @@ angular.module('contactsApp')
 					}
 				});
 			}
-			// No contact previously loaded, let's load the first of the list if not in mobile mode
-			if(ctrl.loading && $(window).width() > 768) {
-				ctrl.setSelectedId(ctrl.contactList[0].uid());
-			}
+			var firstNames = ctrl.contactList.slice(0, 20).map(function (c) { return c.displayName(); });
+			ContactService.getFullContacts(firstNames);
```

Comment:
We still don't want to load the first contact in mobile mode. 
We need a condition to prevent that like we had before ;)